### PR TITLE
Fix(stackdriver): Dont add display names to metric descriptors by def…

### PR DESCRIPTION
**Pull request by ewiseblatt**
@jtk54

### CODE REVIEW

File: spinnaker-monitoring-daemon/spinnaker-monitoring/stackdriver_descriptors.py [hunk 1]
```diff
@@ -311,6 +311,9 @@ def __init__(self, stackdriver, spectator_response_processor):
         'summarize_compound_kinds')
     self.__distributions_also_have_count = stackdriver.stackdriver_options.get(
         'distributions_also_have_count')
+    # When you are familiar with the internal metrics, using the display name
+    # get's confusing so let's require that to be explicitly enabled.
+    self.__add_display_name = stackdriver.stackdriver_options.get('add_display_name')
```
Comment (jtk54) [inline spinnaker-monitoring-daemon/spinnaker-monitoring/stackdriver_descriptors.py]: Does this need to be added as a flag or does this flag already exist?

Comment (ewiseblatt) [reply spinnaker-monitoring-daemon/spinnaker-monitoring/stackdriver_descriptors.py]: It's not intended as a flag, just in spinnaker-monitoring.yml.

The "stackdriver_options" is actually nested in "stackdriver", not the root level.

**Comment by ewiseblatt** [comment]
@spinnakerbot cherry-pick 1.13

**Comment by ewiseblatt** [comment]
@spinnakerbot cherry-pick 1.13

**Comment by jtk54** [comment]
Why was this cherry picked more than once? I think this happened for a few other PRs as well.

**Pull request merged**